### DISCUSSION TITLE
I would like to run cellchat on my Paired scATAC-seq and scRNA-seq Archr Analysis but I don't know how to extract a seurat object from my Arhrproject.

### DISCUSSION THREAD

**Discussion by midichlorianrich** [GreenleafLab/ArchR#1910] [General]
Hi Ryan,

Love Archr. I went through the entire pipeline with my multiome  paired scRNAseq and ATACseq data on 11 different samples. 
Now I have clusters and UMAPs generated via LSI-combined and harmony and I would like to run cellchat on them.
But I have no idea how to extract the RNAseq seurat object from my Archr Project.
Is this possible and if so how would I do it?
This would help me a lot.

thank you!

**Comment by rcorces** [comment]
You cannot extract a Seurat object but you can extract a SummarizedExperiment that could be converted to a Seurat object.
https://www.archrproject.com/reference/getMatrixFromProject.html

**Reply by midichlorianrich** [reply]
Hi Ryan,

I have tried to do this but I am not able to it looks like. 

This is what I tried


proj_ge <- getMatrixFromProject(projmerg, useMatrix = "GeneExpressionMatrix")

library(SingleCellExperiment)

proj_sce <- as(proj_ge, "SingleCellExperiment")

counts <- assay(proj_sce, "GeneExpressionMatrix")

counts(proj_sce) <- counts

libsizes <- colSums(counts)
size.factors <- libsizes/mean(libsizes)
logcounts(proj_sce) <- log2(t(t(counts)/size.factors) + 1)

proj_sce <- as.Seurat(proj_sce, counts = "counts", data = "logcounts")

But this last step fails probably because there are no row names and I don't know what they should be.

Any suggestions?

Thank you!
